wdn/templates_5.0/js/main.js

```diff
@@ -7,3 +7,51 @@ requirejs.config({
 		}
 	}
 });
+
+require(['jquery'], function($) {
+	$(document).ready(function() {
+		var menu_shown_at_small = false;
+		var unl_primary_nav = $('#unl-primary-nav');
+
+		$('#dcf-menu-toggle').click(function (click) {
```

Comment:
A few things re-accessibility:
1) the button to toggle should have an `aria-expanded` attribute
2) the when the nav is opened, focus should be sent to the first link
3) when nav is closed, focus should be sent back to the button that opened it
4) closing on escape might also be helpful (especially for mobile nav when a bluetooth keyboard is being used)

You might be able to easily implement this with the dropdown widget JS I made for the IdM widget. https://github.com/mfairchild365/wdntemplates/blob/e8305700a3a3c5459f1297108a58c644819853bf/wdn/templates_5.0/js/dropdown-widget.js